python/test/server/test_server_tools.py

```diff
@@ -0,0 +1,92 @@
+#    test_server_tools.py
+#        Test various tools for the Python server application
+#
+#   - License : MIT - See LICENSE file.
+#   - Project : Scrutiny Debugger (github.com/scrutinydebugger/scrutiny)
+#
+#   Copyright (c) 2021-2022 scrutinydebugger
+
+import unittest
+from scrutiny.server.server_tools import Throttler
+import time
+
+
+class TestThrottler(unittest.TestCase):
+    def test_throttler_behavior(self):
+        throttler = Throttler(mean_bitrate=1024, window_size_sec=1)
```

Comment:
```suggestion
        petittrot = Throttler(mean_bitrate=1024, window_size_sec=1)
```

:D

okai just kidding.

Reply:
APPROVED